crates/common_enums/src/enums.rs

```diff
@@ -231,6 +232,30 @@ pub enum AuthenticationType {
     NoThreeDs,
 }
 
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Default,
+    Eq,
+    PartialEq,
+    serde::Serialize,
+    serde::Deserialize,
+    strum::Display,
+    strum::EnumString,
+    frunk::LabelledGeneric,
```

Comment:
is this derive required?

Reply:
> is this derive required?
Because of this comment by @lsampras  we changed the location of enums to `common_enums`
https://github.com/juspay/hyperswitch/pull/4880#discussion_r1637501479